cmake/Modules/FindCUDNN.cmake

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2019, Facebook, Deepmind Technologies, NYU, NEC Laboratories
```

Comment:
We have to check if this license is suitable for an Apache project. Please open a jira request with Apache legal to get a confirmation before proceeding.

Reply:
This is BSD license. Does it require approval?
If so, it may be easier to rewrite the file.

Reply:
https://www.apache.org/legal/resolved.html#category-a

BSD should be fine